tests/integration/components/guide-section/subsection/component-test.js

```diff
@@ -22,7 +22,7 @@ module('Integration | Component | guide-section/subsection', function(hooks) {
     `);
 
     assert.dom('[data-test-field="Subsection Title"]')
-      .hasText('§ Warning: Subsection title not found');
+      .hasText('§ Warning: Subsection title not found Edit Translation');
```

Comment:
**thought:**

From this assertion as well as a few below, I wonder if we should move the link outside of the `<h3>` element. (No action needed here. Just a thought.)